### PR TITLE
Fix RWC runner so that it will read default library file from correct location

### PR DESCRIPTION
**Pull request by yuit**
@mhegazy @vladima for code-review

### CODE REVIEW

File: src/harness/rwcRunner.ts [hunk 1]
```diff
@@ -96,9 +105,6 @@ module RWC {
                                 if (useCustomLibraryFile) {
```
Comment (DanielRosenwasser) [inline src/harness/rwcRunner.ts]: Shouldn't you move this up there as well?

File: src/harness/harness.ts [hunk 2]
```diff
@@ -1408,9 +1408,6 @@ module Harness {
                 return diagnostic.file && diagnostic.file.fileName.indexOf("test262-harness") >= 0;
             });
 
-            // Verify we didn't miss any errors in total
-            assert.equal(totalErrorsReported + numLibraryDiagnostics + numTest262HarnessDiagnostics, diagnostics.length, "total number of errors");
```
Comment (mhegazy) [inline src/harness/harness.ts]: why is this removed?

File: src/harness/rwcRunner.ts [hunk 3]
```diff
@@ -65,6 +65,15 @@ module RWC {
                     opts.options.noEmitOnError = false;
                 });
 
+                if (!useCustomLibraryFile) {
+                    let defaultLibPath = ts.sys.resolvePath("built/local/lib.d.ts");
+                    let defaultLib = {
+                        unitName: ts.normalizePath(defaultLibPath),
+                        content: Harness.IO.readFile(defaultLibPath)
```
Comment (mhegazy) [inline src/harness/rwcRunner.ts]: we are reading this file for every test. the file did not change, we should cache it.

File: src/harness/rwcRunner.ts [hunk 4]
```diff
@@ -96,9 +105,6 @@ module RWC {
                                 if (useCustomLibraryFile) {
                                     inputFiles.push(getHarnessCompilerInputUnit(fileRead.path));
                                 }
-                                else {
-                                    inputFiles.push(Harness.getDefaultLibraryFile());
```
Comment (mhegazy) [inline src/harness/rwcRunner.ts]: not sure i understand why this fails.

Comment (yuit) [inline src/harness/rwcRunner.ts]: This fails because Harness.getDefaultLibraryFile will call into ts.sys (which is swap to use data Playback in runWithIOLog) that looks into log array (i,e array representation of the input json) so it will fail to find default lib.d.ts

**Comment by yuit** [comment]
Address in #4488
